tasks/revision-dir.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "Get latest and current revision"
+  git:
+    repo: "{{ atom_repository_url }}"
+    version: "{{ atom_repository_version }}"
+    dest: "{{ atom_path }}/{{ atom_revision_directory_latest_symlink_dir }}" 
+    clone: no
+    update: no
+  register: "atom_revision_git_output"
+  when: "atom_revision_directory|bool and atom_revision_directory_latest_symlink_dir is defined"
+
```

Comment:
Does this git task with `clone:no` and `update:no` work even when doing the first deployment (a new deployment)? (From ansible docs: "If clone: no, do not clone the repository even if it does not exist locally". "If update:no, do not retrieve new revisions from the origin repository")